<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyDialog</class>
 <widget class="QDialog" name="MyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>215</width>
    <height>228</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>440</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Connect</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget { background-color: rgba(32, 43, 47, 60%); }
QWidget#widget { background-color: rgba(32, 43, 47, 0%); }
QLabel {  background-color: rgba(32, 43, 47, 0%); color: rgb(255, 255, 255); }
QLineEdit {
    background-color: #202B2F;
	color: #FFFFFF;
    border-style: outset;
    border-width: 2px;
    border-radius: 3px;
    border-color: beige;
    font: bold 10px;
}

</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Leelawadee UI</family>
       <pointsize>14</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;Leelawadee UI&quot;;</string>
     </property>
     <property name="text">
      <string>Connect to Host</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="5" column="1">
          <widget class="QLineEdit" name="InputIP">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">border-radius:0px;
border:1px solid #fff;</string>
           </property>
           <property name="maxLength">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Your Name (max 10 chars)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLabel" name="label_5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Port</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QLineEdit" name="InputPort">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>45</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-radius:0px;
border:1px solid #fff;</string>
           </property>
           <property name="text">
            <string>1234</string>
           </property>
           <property name="maxLength">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Host IP</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="6" column="1" colspan="2">
          <widget class="QPushButton" name="pushButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>154</width>
             <height>0</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton#pushButton {
    background-color: #8BB2DA;

    font: bold 12px;
    min-width: 10em;
    padding: 6px;
}
QPushButton#pushButton:pressed {
    background-color: rgb(197,216,236);
	border-style: inset;
 
}</string>
           </property>
           <property name="text">
            <string>Login</string>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
           <property name="autoDefault">
            <bool>true</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="QComboBox" name="professionComboBox">
           <property name="styleSheet">
            <string notr="true">QComboBox {
    background-color: #8BB2DA;
	color: rgb(255, 255, 255);
    border: 1px solid #AAA;
	font-size: 14px;
}
QComboBox:editable {
    background: #8BB2DA;
	color: rgb(255, 255, 255);
}

QComboBox:on { /* shift the text when the popup opens */
    padding-top: 3px;
    padding-left: 4px;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
	color: rgb(255, 255, 255);
	border:none;


}
QComboBox::down-arrow:on { /* shift the arrow when popup is open */
    top: 1px;
    left: 1px;
}
QComboBox QAbstractItemView {
    border: 2px solid darkgray;
    selection-background-color:#8BB2DA;
	color: rgb(255, 255, 255);
}</string>
           </property>
           <property name="editable">
            <bool>false</bool>
           </property>
           <property name="currentText">
            <string>Select Profession</string>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <property name="maxVisibleItems">
            <number>10</number>
           </property>
           <item>
            <property name="text">
             <string>Select Profession</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Elementalist</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Engineer</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Guardian</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Mesmer</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Necromancer</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Ranger</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Revenant</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Thief</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Warrior</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QLineEdit" name="InputName">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">border-radius:0px;
border:1px solid #fff;</string>
           </property>
           <property name="maxLength">
            <number>15</number>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
